client/galaxy/scripts/mvc/tool/tool-form-base.js

```diff
@@ -228,10 +238,28 @@ define(['utils/utils', 'utils/deferred', 'mvc/ui/ui-misc', 'mvc/form/form-view',
                     tooltip : 'Download this tool',
                     onclick : function() {
                         window.location.href = Galaxy.root + 'api/tools/' + options.id + '/download';
-                    }
+                 }
                 });
             }
 
+            // add tours option
+            if (options.tours && options.tours.length > 0) {
+                for (var i in options.tours) {
+                    var tour = options.tours[i];
+                    tours_button.addMenu({
+                        title   : tour.name,
+                        tour    : tour,
+                        tooltip : 'Start an interactive tool introduction',
+                        icon    : 'fa-cube',
+                        onclick : function() {
+                            window.location.href = Galaxy.root + 'api/tools/' + options.id + '/tours/' + tour.tour_id;
```

Comment:
So the neat thing here is that we actually don't need the URL, with the way we've built the javascript objects.  We'll just need to include the tours javascript here and do Tours.start_tour(tour_id) and it'll go, without any redirect.  I'll try to mock up a quick version of this to show.